website/docs/reference/resource-configs/hard-deletes.md

```diff
@@ -0,0 +1,113 @@
+    +hard_deletes: "ignore", "invalidate", or "new_record"
+```
+
+</File>
+
+<File name='snapshots/<filename>.sql'>
+
+```sql
+{{
+    config(
+        unique_key='id',
+        strategy='timestamp',
+        updated_at='updated_at',
+        hard_deletes='ignore', 'invalidate', 'new_record'
+    )
+}}
+```
+
+</File>
+
+
+## Description
+
+Use the `hard_deletes` configuration to track hard deletes by adding a new record when row become "deleted" in source. 
+Replaces the `invalidate_hard_deletes` config to give you more control on how to handle deleted rows from the source.
+
+import HardDeletes from '/snippets/_hard-deletes.md';
+
+<HardDeletes />
+
+:::warning
+
+If you're updating an existing snapshot to use the `hard_deletes` config, dbt _will not_ handle migrations automatically. We recommend either only using these settings for net-new snapshots, or arranging an update of pre-existing tables before enabling this setting.
+:::
+
+## Default
+
+By default, if you don’t specify `hard_deletes`, it'll automatically default to `ignore`. Deleted rows will not be tracked and their `dbt_valid_to` column remains `NULL`.
+
+The `hard_deletes` config has three options:
+
+| Field | Description |
+| --------- | ----------- |
+| `ignore` (default) | No action for deleted records. |
+| `invalidate` | Behaves the same as the existing `invalidate_hard_deletes=true`, where deleted records are invalidated by setting `dbt_valid_to`. |
+| `new_record` | Tracks deleted records as new rows using the `dbt_is_deleted` meta field when records are deleted.|
+
+## Impact on snapshot records
+
+- **Backward compatibility**: The `invalidate_hard_deletes` config is still supported for existing snapshots but can't be used alongside `hard_deletes`.
+- **New snapshots**: For new snapshots, we recommend using `hard_deletes` instead of `invalidate_hard_deletes`.
+- **Migration**: If you switch an existing snapshot to use `hard_deletes` without migrating your data, you may encounter inconsistent or incorrect results, such as a mix of old and new data formats.
+
+## Example
+
+<File name='snapshots/schema.yml'>
+
+```yaml
+snapshots:
+  - name: my_snapshot
+    config:
+      hard_deletes: new_record  # options are: 'ignore', 'invalidate', or 'new_record'
+      strategy: timestamp
+      updated_at: updated_at
+    columns:
+      - name: dbt_valid_from
+        description: Timestamp when the record became valid.
+      - name: dbt_valid_to
+        description: Timestamp when the record stopped being valid.
+      - name: dbt_is_deleted
+        description: Indicates whether the record was deleted.
+```
+
+</File>
+
+The resulting snapshot table contains the `hard_deletes: new_record` configuration. If a record is deleted and later restored, the resulting snapshot table might look like this:
+
+| id | dbt_scd_id           |   Status | dbt_updated_at       |   dbt_valid_from    |     dbt_valid_to     | dbt_is_deleted | 
```

Comment:
same comment as earlier example - the only thing that should change about the record is it being deleted then restored